mysql-ballerina/client.bal

```diff
@@ -149,12 +149,24 @@ type ClientConfiguration record {|
 # + useXADatasource - Boolean value to enable XADatasource
 # + connectTimeout - Timeout (in seconds) to be used when connecting to the mysql server
 # + socketTimeout - Socket timeout (in seconds) during the read/write operations with mysql server,
-#                            0 means no socket timeout
+#                   0 means no socket timeout
+# + connectionAttributes - A comma-delimited list of user-defined key:value pairs (in addition to standard
+#                          MySQL-defined key:value pairs) to be passed to MySQL Server
+# + useConfigs - Load the comma-delimited list of configuration properties before parsing the URL or applying
+#                user-specified properties
```

Comment:
IMO. This is not the correct way, asking the user to provide properties as `key:value` with comma-separated will cause errors at runtime, if there is a typo in provided values

Reply:
connectionAttributes option can be used to pass user-defined key values pairs. so I think we can keep this option because the key:value is defined by the user.

Reply:
Can the user pass any key value pair?

Reply:
You can find the defined key value pairs used when creating the connection in this [documentation](https://dev.mysql.com/doc/refman/8.0/en/connecting-using-uri-or-key-value-pairs.html)

Reply:
How application-specific connection attributes work? DB server also needs to aware of those custom properties right?

Reply:
yes, DB server is aware of those custom properties. according to the mysql documentation those connection attributes are saved in [performance Schema tables](https://dev.mysql.com/doc/refman/8.0/en/performance-schema-connection-attribute-tables.html).

Reply:
removed `connectionAttributes` and `useConfigs` options in https://github.com/ballerina-platform/module-ballerinax-mysql/pull/123/commits/7706cbcd32c09180551cdac2920756e92a1069b1